src/main/java/Duke.java

```diff
@@ -1,10 +1,282 @@
+import duke.task.Task;
+import duke.task.ToDo;
+import duke.task.Event;
+import duke.task.Deadline;
+import duke.exceptions.DukeException;
+import duke.exceptions.EmptyInputException;
+import duke.exceptions.IllegalInputException;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+    public static final String LINE = "____________________________________________________________\n";
+    public static final String BYE_MESSAGE = "Bye. Hope to see you again soon!\n";
+    public static boolean canExit = false;
+
+    /**
+     * Prints message after adding To-Do/Deadline/Event duke.task
+     *
+     * @param myList
+     */
+    public static void printTaskMessage(ArrayList<Task> myList) {
+        System.out.println(LINE
+                + "Got it. I've added this duke.task:\n"
+                + myList.get(myList.size() - 1) + "\n"
+                + "Now you have " + myList.size() + " in the list.\n"
+                + LINE);
+    }
+
+    /**
+     * Prints a greeting user message
+     */
+    public static void printGreetMessage() {
+        System.out.println(LINE + "Hello! I'm Duke\n"
+                + "What can I do for you?\n"
+                + LINE);
+    }
+
+    /**
+     * Prints exiting message before program stops
+     */
+    public static void printByeMessage() {
+        System.out.println(BYE_MESSAGE);
+    }
+
+    /**
+     * Prints an added message
+     *
+     * @param myList
+     */
+    public static void printAddedTaskMessage(ArrayList<Task> myList) {
+        System.out.println(LINE + "added: "
+                + myList.get(myList.size() - 1).getDescription()
+                + "\n" + LINE);
+    }
+    public static void printIllegalInputMessage() {
+        System.out.println(LINE
+                +  "☹ OOPS!!! I'm sorry, but I don't know what that means :-(\n"
+                + LINE);
+    }
+    public static void printEmptyInputMessage(String task) {
+        System.out.println(LINE
+                + "☹ OOPS!!! The description of " + task + " cannot be empty.\n"
+                + LINE);
+    }
+    /**
+     * Adds the input text into the list
+     *
+     * @param s
+     * @param myList
+     */
+    public static void addList(String s, ArrayList<Task> myList) {
```

Comment:
Maybe you can rename your `String s` to something that is more informative to readers.

Reply:
I do see this issue coming up further down in your code. So do take note and update them!